game/client_types/autoplay.js

```diff
@@ -24,7 +24,7 @@ module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
     stager.extendAllSteps(function(o) {
         o._cb = o.cb;
         o.cb = function() {
-            var _cb, stepObj, id;
+            let _cb, stepObj, id;
```

Comment:
This code is executed in the browser (it is the `cb` function), can you revert to `var`?

Reply:
ok, fixed.